website/docs/plugin/framework/handling-data/attributes.mdx

```diff
@@ -468,6 +468,308 @@ attribute value in a way that Terraform will understand. `attr.Value` has the
 following methods.
 
 | Method             | Description                                                                                                                                                                                                                     |
-| ------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+|--------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | `ToTerraformValue` | Returns a Go type that is valid input for [`tftypes.NewValue`](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-go/tftypes#NewValue) for the `tftypes.Type` specified by the `attr.Type` that creates the `attr.Value`. |
 | `Equal`            | Returns true if the passed attribute value should be considered to the attribute value the method is being called on. The passed attribute value is not guaranteed to be of the same Go type.                                   |
+
+## Terraform Configuration and Schema
+
+### Simple Attributes
```

Comment:
The "Framework Types and Value Types" above walks through many of these Simple/Collection configuration and schema examples although it looks like they need to be updated for `schema.XXXAttribute` syntax (which I'll handle separately).

Reply:
Ok. I'll remove these, I just thought there might be some value in being able to quickly scan the TF configuration and the schema to see the correlation for simple attributes between the two.